doc/steps.md

```diff
@@ -591,11 +592,32 @@ Input variables from config:
   (`.tarantool.cookie` will be kept independently of this variable);
   it's possible to use bash patterns, e.g. `*.control`.
 - `cartridge_restore_backup_path` - path to the instance backup archive on the remote machine;
+- `cartridge_restore_backup_path_local` - path to the instance backup archive on the local machine;
 - `cartridge_remote_backups_dir` - directory with backups on the remote;
 - `cartridge_force_restore` - flag indicates that conflicting files should be overwritten;
 - `cartridge_allow_alien_backup` - flag indicates that backup of instance with another name can be used;
 - `cartridge_skip_cleanup_on_restore` - flag indicates that cleanup before restoring should be skipped.
 
+### Step `backup_files`
```

Comment:
As for me, step name doesn't reflect the main idea that stopped instances are backed up.